event/src/unvalidated/event.rs

```diff
@@ -239,6 +239,23 @@ where
             )),
         }
     }
+
+    /// Get the CID of the init event of the stream for this event
+    pub fn stream_cid(&self) -> &Cid {
```

Comment:
This method already exists verbatim above but as the the `id()` method. I like the `stream_cid` name better but we should only have one method.

Reply:
I will remove id()